packages/sdk-react/src/components/payables/PayableDetails/PayableDetails.test.tsx

```diff
@@ -609,5 +611,64 @@ describe('PayableDetails', () => {
         ).rejects.toThrowError(/Unable to find an element/);
       });
     });
+
+    describe('isInvoiceOverdue', () => {
+      const createInvoice = (
+        status: components['schemas']['PayableStateEnum'],
+        due_date: components['schemas']['PayableResponseSchema']['due_date']
+      ): components['schemas']['PayableResponseSchema'] => ({
+        id: 'some-unique-id',
+        created_at: new Date().toISOString(),
+        updated_at: new Date().toISOString(),
+        entity_id: 'some-entity-id',
+        status,
+        due_date,
+        payable_origin: 'upload',
+        source_of_payable_data: 'user_specified',
+      });
+
+      const testCases = [
+        {
+          description: 'should return false when due_date is after today',
+          status: PayableStateEnum.WAITING_TO_BE_PAID,
+          due_date: new Date(Date.now() + 48 * 60 * 60 * 1000).toISOString(),
+          expected: false,
+        },
+        {
+          description: 'should return false when due_date is today',
+          status: PayableStateEnum.WAITING_TO_BE_PAID,
+          due_date: new Date().toISOString(),
```

Comment:
This will return string with current timestamp. Therefore check will work in tests, but will fail in real world.
```
  if (new Date(invoice.due_date).getTime() === new Date().getTime())
    return false;
```
You need test dates without time in them.